### ISSUE TITLE
Status of "let" block?

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap--Build-Your-Own-Blocks#2153]
I see that the pixels library defines a 'let' block. I'm guessing I'm not the only one who prefers it to using 'script variables' followed by 'set'. So why is it hidden away in the pixel library rather than either being in 'tools' or being in the core (the variables palette)?

**Comment by brianharvey** [comment]
Right you are.  I just haven't gotten around to it.  Thanks for the reminder!

**Comment by jmoenig** [comment]
yeah, hold it, Brian, let's talk about this first. I have some more ideas for dealing with variables...
